source/iomap_otbm.cpp

```diff
@@ -1064,7 +1103,7 @@ bool IOMapOTBM::loadSpawns(Map& map, pugi::xml_document& doc)
 			if(creatureTile->getLocation()->getSpawnCount() == 0) {
 				// No spawn, create a newd one
 				ASSERT(creatureTile->spawn == nullptr);
-				Spawn* spawn = newd Spawn(5);
+				Spawn* spawn = newd Spawn(1);
```

Comment:
Spawn size to 1 (I see no sense in being 5, since a creature should occupy the smallest possible spawn, to prevent it from going too far from its location)